packages/cozy-client/src/devtools/index.jsx

```diff
@@ -103,7 +103,9 @@ const DevToolsPanel = props => {
           />
         </ListGridItem>
         {panels.map(panelOptions =>
-          currentPanel === panelOptions.id ? <panelOptions.Component /> : null
+          currentPanel === panelOptions.id ? (
+            <panelOptions.Component key={panelOptions.id} />
```

Comment:
Ah there it was, I saw a warning at some point for this key= and could not find the missing key 😅

Reply:
I admit, it was not the easiest to find!